great_expectations/datasource/fluent/file_path_data_asset.py

```diff
@@ -172,6 +173,7 @@ def build_batch_request(
         self,
         options: Optional[BatchRequestOptions] = None,
         batch_slice: Optional[BatchSlice] = None,
+        partitioner: Optional[AbstractPartitioner] = None,
```

Comment:
How are aliases parsed with `@public_api`?

Reply:
that's an interesting question - I'm not sure